src/transcribo_backend/services/summarization_service.py

```
@@ -0,0 +1,44 @@
+        self.config = config
+
+    async def summarize(self, transcript: str) -> Summary:
+        """
+        Summarize a transcript of a meeting.
+        """
+        try:
+            client = AsyncOpenAI(api_key=self.config.api_key, base_url=self.config.api_key)
+            models = await client.models.list()
+
+            if not models.data:
+                raise ValueError("Could not find any models available from the API")
```

Ruff (0.14.11)

Abstract `raise` to an inner function

(TRY301)